src/api/Annotations.js

```diff
@@ -70,14 +75,23 @@ export default class Annotations extends MarkerBasedApi {
         });
     }
 
-    updateAnnotation(
-        fileId: string,
+    updateAnnotation({
```

Comment:
I found few other examples when we passing object as a parameter (eg. Comments.js). I think in this use case, with 7 parameters, including 2 optional ones, it would help with readability.

Reply:
Perhaps, but we have other many other methods, even in this file (i.e., `getAnnotations`) that use quite a few parameters; we should keep our signatures consistent. Would moving `message` and `status` into a `payload` object help?